adapters/kelp_gain_linea/src/index.ts

```diff
@@ -109,6 +114,27 @@ export const getUserTVLByBlock = async (blocks: BlockData) => {
     });
   });
 
+  let totalRsEthSaveToCSV = csvRows.reduce(
+    (acc, s) => acc + s.token_balance,
+    0n
+  );
+
+  const rsEthTotalSupply = await rsETHTotalSupply(blockNumber);
+
+  console.log(
+    `TOTAL rsEth balance  in CSV : ${ethers.utils.formatEther(
+      totalRsEthSaveToCSV.toString()
+    )}\nTOTAL rsETH supply linea: ${ethers.utils.formatEther(
+      rsEthTotalSupply.toString()
+    )}`
+  );
+
+  if (totalRsEthSaveToCSV > rsEthTotalSupply) {
+    throw new Error(
+      `The total balance in CSV ${totalRsEthSaveToCSV} can not more than total supply ${rsEthTotalSupply}`
+    );
+  }
```

Comment:
currently this is just checking if sum of rseth in csv is  more than total supply. but my concern here is that one of the function might return 0 result when it is not suppose to, which we ended up ingesting that. 

can we modify this check to make sure those function that is suppose to return data in the specific block is doing so, and throw error if it isnt?